internal/widget/scroller.go

```diff
@@ -393,13 +393,16 @@ func (s *Scroll) CreateRenderer() fyne.WidgetRenderer {
 
 // ScrollToBottom will scroll content to container bottom - to show latest info which end user just added
 func (s *Scroll) ScrollToBottom() {
+	if s.Content.MinSize().Height > s.Size().Height {
+		s.Offset.Y = s.Content.MinSize().Height - s.Size().Height
+		s.Base.Refresh()
```

Comment:
This fix doesn't seem right - making a change then refreshing but then also making another change.

Really it should call `scrollBy` so the fix should be revising the parameters to that method.
And that `Refresh` looks like it's not needed any more.

Reply:
You should also save the result of `MinSize()` and `Size()` instead of computing it multiple times in succession.